src/server/editorServices.ts

```diff
@@ -1350,6 +1350,22 @@ namespace ts.server {
             }
         }
 
+        /*@internal*/
+        onExtendedConfigChangedForConfiguredProject(project: ConfiguredProject, extendedConfigFile: string) {
+            const configFileExistenceInfo = this.getConfigFileExistenceInfo(project);

```

Comment:
i dont think we want existence info for extended config files..  the existence info is used for inferred projects.. so plain and simple map of watches is what you want instead.